semeio/jobs/design2params/design2params.py

```diff
@@ -181,22 +181,27 @@ def _read_excel(file_name, sheet_name, header=0):
     """
     try:
         return pd.read_excel(file_name, sheet_name, header=header, dtype=str)
-    except IOError:
-        raise SystemExit("File {} not found".format(file_name))
+    except IOError as err:
+        raise SystemExit("File {} not found".format(file_name)) from err
     except Exception as err:
         raise SystemExit(
             (
                 "File {} is probably not of correct type. Failed with exception '{}'"
             ).format(file_name, str(err))
-        )
+        ) from err
 
 
 def _validate_design_matrix_header(design_matrix):
     """
     Validate header in user inputted design matrix
     :raises: SystemExit if design matrix contains empty headers
     """
-    unnamed = design_matrix.loc[:, design_matrix.columns.str.contains("^Unnamed")]
+    try:
+        unnamed = design_matrix.loc[:, design_matrix.columns.str.contains("^Unnamed")]
+    except (AttributeError, ValueError) as err:
```

Comment:
Let's add a comment stating why we're catching these two errors, as they originally stem from the same issue (or close to?). Never the less, whenever we stop supporting pandas < 1.0, we only need to catch one of them.